speaker_verification/__init__.py

```diff
@@ -50,7 +50,7 @@ def validate_user(args):
         Commandline arguments passed with argparse. Required arguments are args.id and args.audio_path
     """
     validate_id(args.id)
-    user_row = select_db_row(args.db_table, args.id)
+    user_row = select_db_row(args.db_table, args.id,args.database_path)
```

Comment:
```suggestion
    user_row = select_db_row(args.db_table, args.id, args.database_path)
```
Please use an autoformatter with a line length argument of 90 characters like `black` in the controbutions guideline. It'll save time in reviewing changes for small formatting changes like this.